### PR TITLE
📖 add Cluster API release-1.8 timeline document 

### LINKED ISSUE #9978: Incorporate additional patch release into next release cycle planning (v1.8)
- update release documentation to incorporate the additional patch release

pending approval/community consensus on: https://github.com/kubernetes-sigs/cluster-api/pull/9971

/area release

### PR DESCRIPTION
**Pull request by cahillsf**
**What this PR does / why we need it**:
Adds preliminary schedule for `v1.8` minor release

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #9978

/area release
/kind documentation

### CODE REVIEW

**Review by fabriziopandini — COMMENTED** [review]
Thanks for adding the path release faster Kubernetes 1.30 support!

The release calendar for 1.8 seems ok, but I would like to bring to the attention that by playing a quick simulation, the schedule for 1.9 will clash with xMas holidays. 

What do you think about shortening 1.8 release cycle by 1 week (16 weeks from W17 to W33), then we can plan 1.9 to be more or less the same + KubeCon idle week (17 weeks from W33 to W50)?
This will place 1.9 release around the 10th December with 1 week of buffer before the xMas break...

File: docs/release/releases/release-1.8.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# Cluster API v1.8
+
+## Timeline
+
+The following table shows the preliminary dates for the `v1.8` release cycle.
+
+| **What**                                             | **Who**      | **When**                    | **Week** |
+|------------------------------------------------------|--------------|-----------------------------|----------|
+| Start of Release Cycle                               | Release Lead | Monday 22nd April 2024      | week 1   |
+| *v1.6.x & v1.7.x released* <sup>[1]</sup>            | Release Lead | Tuesday 23rd April 2024     | week 1   |
```
Comment (sbueringer) [inline docs/release/releases/release-1.8.md:10]: For Kubernetes we only need v1.7.x. So I would suggest to drop v1.6 here (as we don't cherry-pick 1.30 support into 1.6)

**Comment by sbueringer** [comment]
Just to avoid misunderstandings. I assume you mean basically move "first "standard" patch release" and everything below up by 1 week?

Assuming I got it correct so far. I think I would move it simply up by 2.

This will give us only 3 weeks between v1.7.1 and v1.7.2. But we'll then still have 5 weeks between v1.5.0/v1.6.0/v1.7.0 (16h April) and v1.6.1/v1.7.2 (21st May). Given that v1.7.1 is an additional release we should end up with ~ the following time between patch releases 4 weeks / 5 weeks / 4 weeks / 4 weeks / ...

**Comment by fabriziopandini** [comment]
What about keeping mothly patch releases where they are (), and cut v1.8.0-beta.0 week 12 instead of 14, shiting up by two everything else

**Comment by sbueringer** [comment]
> It moves up by one; this is how it should look like for me

I think @cahillsf just asked where to trim the second week. We have consensus on trimming two weeks (I think)

**Comment by cahillsf** [comment]
nice thanks for the input and clarification 😄 . just pushed up the recommended changes

**Comment by sbueringer** [comment]
Thank you very much!

/lgtm

/assign @fabriziopandini

**Review by chrischdi — COMMENTED** [review]
/lgtm

**Comment by fabriziopandini** [comment]
/approve

Thanks @cahillsf for driving this effort!

**Pull request merged**